server/neptune/workflows/internal/terraform/runners/run_step_runner.go

```diff
@@ -0,0 +1,102 @@
+package runners
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/hashicorp/go-version"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/activities"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/github"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/steps"
+	"go.temporal.io/sdk/workflow"
+)
+
+type ExecuteCommandActivities interface {
+	ExecuteCommand(context.Context, activities.ExecuteCommandRequest) activities.ExecuteCommandResponse
+}
+
+const planfileSlashReplace = "::"
+
+type RunStepRunner struct {
+	Activity ExecuteCommandActivities
+}
+
+func GetPlanFilename(projName string) string {
```

Comment:
I want to refrain from using project at all here. This gets confusing with our internal definition of project at lyft.  I think root is a better name going foward. Pls change all instances to that.

Reply:
Gotcha!